### DISCUSSION TITLE
Spatie Translatable Plugin causes error on image field

### DISCUSSION THREAD

**Discussion by nurmuhammet-ali** [filamentphp/filament#2312] [Help]
### Package

filament/filament

### Package Version

v2.10.45

### Laravel Version

v9.8.1

### Livewire Version

v2.10.5

### PHP Version

8.1

### Bug description

Since I added Spatie Translatable Plugin, the image uploader started working incorrectly. Every time I switch language, the image updates and not saved on db. Then eventually, app throws an exception. Note that my model is unguarded. Very small demo:

https://user-images.githubusercontent.com/25463082/163874709-a09453b9-1bb8-453c-ad59-ef603633e96c.mp4

**Comment by danharrin** [comment]
Hey, I cannot reproduce this
